pre_commit/color.py

```diff
@@ -48,6 +48,9 @@ def use_color(setting):
     if setting not in COLOR_CHOICES:
         raise InvalidColorSetting(setting)
 
+    if 'NO_COLOR' in os.environ:
```

Comment:
this isn't quite right -- `NO_COLOR= git commit ...` should still have color

Reply:
@asottile Ok, I pushed a new commit